vscode-extension/src/utilities/pythonSetupUtils.ts

```diff
@@ -354,7 +354,7 @@ export async function savePythonInterpreterToCache(): Promise<void> {
   await config.update(
     PYTHON_INTERPRETER_CACHE_KEY_NAME,
     pythonPath,
-    vscode.ConfigurationTarget.Workspace
+    getConfigurationTarget()
```

Comment:
We also need to update the read path from https://github.com/lastmile-ai/aiconfig/blob/6aff52c59b056ab30bd35087f970a9fa99f9059b/vscode-extension/src/utilities/pythonSetupUtils.ts#L389-L390 for this

Reply:
Ok nm, just read this here that we already check these in the user global settings first, and they will be overridden by the smaller-scoped settings by default
<img width="1584" alt="Screenshot 2024-02-25 at 01 33 07" src="https://github.com/lastmile-ai/aiconfig/assets/151060367/072a50eb-96cd-47af-8109-23a166978434">